### PR TITLE
[ENT] add missing nodes from the node query

### PR DESCRIPTION
**Pull request by pxp928**
# Description of the PR

The node query did not return nodes for `packageNamespace`, `packageType`, `SourceNamespace`, `SourceType` and `VulnType`. This is now fixed and will properly return for the IDs specified.

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If GraphQL schema is changed, GraphQL client updates/additions have been made
- [ ] If OpenAPI spec is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [x] All CI checks are passing (tests and formatting)
- [x] All dependent PRs have already been merged

### CODE REVIEW

File: pkg/assembler/backends/ent/backend/package.go [hunk 1]
```diff
@@ -467,6 +467,63 @@ func getPkgVersion(ctx context.Context, client *ent.Client, pkgin model.PkgInput
...
+		return nil, fmt.Errorf("failed to get packageNamespace for node ID: %s with error: %w", nodeID, err)
+	}
+
+	if len(pn) > 0 {
+		pkgNamespace := &model.Package{
+			ID:   pkgTypeGlobalID(pn[0].Type),
+			Type: pn[0].Type,
+			Namespaces: []*model.PackageNamespace{
+				{
+					ID:        pkgNamespaceGlobalID(strings.Join([]string{pn[0].Type, pn[0].Namespace}, ":")),
```
Comment (mrizzi) [inline pkg/assembler/backends/ent/backend/package.go:492]: Should `guacIDSplit` be used instead of `:`?

**Review by mrizzi — COMMENTED** [review]
Added a comment just to clarify the `guacIDSplit` usage

File: pkg/assembler/backends/ent/backend/package.go (hunk 1 above)
Comment (pxp928) [reply pkg/assembler/backends/ent/backend/package.go]: yes! Good catch! Thanks!

Comment (pxp928) [reply pkg/assembler/backends/ent/backend/package.go]: fixed

**Review by mrizzi — APPROVED** [review]
@pxp928 LGTM, thanks :+1:

**Review by mihaimaruseac — APPROVED** [review]
Thank you!

**Pull request merged**